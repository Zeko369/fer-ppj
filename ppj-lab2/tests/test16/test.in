OP_PRIDRUZI 1 =
BROJ 1 35
