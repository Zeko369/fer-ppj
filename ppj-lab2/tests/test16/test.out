err OP_PRIDRUZI 1 =
